DatadogRUM/Sources/RUMMonitor/Dependencies/RUMAppLaunchManager.swift

```diff
@@ -0,0 +1,195 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import DatadogInternal
+import Foundation
+
+internal class RUMAppLaunchManager {
```

Comment:
suggestion/ This looks like a scope. Do you think it can be turned into a `AppLaunchVitalScope`, or is there specificities blocking that?